src/Compilers/VisualBasic/Portable/Semantics/Conversions.vb

```diff
@@ -960,11 +960,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Return Nothing 'ConversionKind.NoConversion
             End If
 
-            Dim sourceType As TypeSymbol = source.Type
+            Dim sourceType As TypeSymbol = If(source.Kind = BoundKind.TupleLiteral,
```

Comment:
More complicated logic in array literals are needed mostly because array literals would always have a default type, but that type may not be useful in some scenarios.
For example ```{AddressOf Main}``` would have a default type Object() to which the array cannot convert. Such the type is not useful for some scenarios such as type inference. There are other data on the literal that could be used in inference as well. If we just grab the default type, we would lose all that information, that is why a wrapper symbols is used, and the code is very careful to erase the wrapper when it is done with inference.

So far tuples did not need such complexity. It is permitted for a tuple literal to not have default type, and if it does not have one - so be it, it cannot as a whole source any data into any kind of inference. 
Note that it still can source inference information if there is element-wise conversion. Unlike arrays, tuple elements do not need a dominant type between them.